site/content/entry/citra-progress-report-2023-q1/index.md

```diff
@@ -0,0 +1,413 @@
+
+After experiencing this themselves via accidentally clicking onto another game whilst already in a game, [foghawk](https://github.com/foghawk) added a confirmation check. No longer will you lose any important save data from this issue, and you can carry on playing after selecting one of the two options!
+
+{{< figure src="stopEmu.png"
+    title="No more accidentally losing your progress." >}}
+	
+## Per-game configurations ([#6187](https://github.com/citra-emu/citra/pull/6187), [#6219](https://github.com/citra-emu/citra/pull/6219), [#6224](https://github.com/citra-emu/citra/pull/6224)) by [GPUCode](https://github.com/GPUCode)
+
+Based off of the yuzu implementation of the same name, Per-Game Configurations have landed in Citra! This is part of a series of changes coming to Citra to modernize the frontend of the emulator.
+
+Tired of having to change your settings for each game you play every time you launch it? Well, worry no more! With this change to Citra, each of your games now has their own settings profile. 
+You can change the emulation speed, graphical settings, audio settings such as what audio emulation you’re using, the volume of each game, and many more!
+
+With this PR, the option for **Alternate Speed**, which was available in the general settings of Citra, has been removed in favour of an option called **Per-game Emulation Speed**. This setting works in the exact same was as the previous **Alternate Speed** setting, however it is now found inside of the per-game settings instead of the global settings. The hotkey for this has also been renamed appropriately to reflect the name change of the setting.
+
+Individual config files for each per-game setting you may have are stored in `config -> custom -> <titleid>.ini` These .ini files can be edited using a text editor application, and the changes will appear in Citra.
+
+This is another addition which has been highly requested in the Citra community, and we are proud that it is finally here.
+
+{{< figure src="perGame.png"
+    title="New way to configure your games!" >}}
+	
+### Change Monoscopic Render mode to a dropdown ([#6215](https://github.com/citra-emu/citra/pull/6215)) by [vitor-k](https://github.com/vitor-k)
+
+Just a small QoL update here! Having a dropdown box to change between two settings is pretty clean, don’t you think? 
+Stereoscopic is the default in Citra for 3D capabilities, which displays the 3D to both eyes at the same time. Monoscopic Render emulates the 3DS displaying its 3D capabilities to only one eye at a time. Some games render differently depending on which eye the display is focusing on. The major image differences between the eyes happens at 0% depth, as some games do not bother rendering some elements of in game material to both eyes.
```

Comment:
The default for Citra is stereoscopy off, so it would be monoscopic mode.
I would also change the "one at a time", as that might imply it's alternating the screens (which is what interlaced 3d does).

A suggestion (feel free to change it a bit if necessary):

> The 3ds renders one screen for each eye. The stereoscopic 3d modes will use various methods for displaying each screen to the correct eye, but monoscopic mode just chooses one of them to be displayed. Some games have visual differences depending on which eye is shown. The major image differences between the eyes happens at 0% depth, as some games do not bother rendering some elements of in game material to both eyes.